src/chart/custom/CustomView.ts

```diff
@@ -352,7 +354,36 @@ function createEl(elOption: CustomElementOption): Element {
         el = new graphicUtil.Group();
     }
     else if (graphicType === 'compoundPath') {
-        throw new Error('"compoundPath" is not supported yet.');
+        const shape = (elOption as CustomCompoundPathOption).shape;
+        if (!shape || !shape.paths) {
+            let errMsg = '';
+            if (__DEV__) {
+                errMsg = 'shape.paths must be specified in compoundPath';
+            }
+            throwError(errMsg);
+        }
+        const paths = map(shape.paths as Path[], function (path) {
+            if (path.type === 'path') {
+                return graphicUtil.makePath(path.shape.pathData, path, null);
+            }
+            const Clz = graphicUtil.getShapeClass(path.type);
+            if (!Clz) {
+                if (typeof path.buildPath === 'function') {
```

Comment:
I think probably we should not public that `buildPath`.


1. That function is not officially public in any other place, including not public in custom series path.
2. If publish `buildPath` function, the signature must not be changed or need backward compat when changing. And the parameter of `buildPath` includes `ctx`, which is `PathProxy` or `CanvasContext`. Although they are always accessible never be officially public and promised in API stability.

Is it needed in some scenarios? @Ovilia